### ISSUE TITLE
Not receiving Error rate of 7%, for Depth of 40 and k =12 on CIFAR10

### ISSUE THREAD

**Issue by nbansal90** [tdeboissiere/DeepLearningImplementations#63]
Hello,

I just ran the program provided,for a Depth of 40 and Growth rate of 12. Accordingly, the number of dense blocks were kept as 3, Number of filters for 1st convolution layer was kept as 24(2*k) But still after 300 epochs the test accuracy achieved is 90.78% for CIFAR10, far from what is described in the paper.I have not changed any thing else, could you please let me know, if I am missing something here.

**Comment by tdeboissiere** [comment]
I may have made a mistake when I updated the code to be compatible with keras 2 (esp. if batch norm is involved).

You could try to get back to commit a122bd3fd749645a147e023b5ed7e62aa30927b1 and keras 1.2 and see if that improves things.
